Tests/MockTest.php

```diff
@@ -0,0 +1,77 @@
+<?php
+
+namespace Danhunsaker\Geocoder\Provider\Mock\Tests;
+
+use Danhunsaker\Geocoder\Provider\Mock\Mock;
+use Http\Client\HttpClient;
+use PHPUnit\Framework\TestCase;
+
+class MockTest extends TestCase
+{
+    public function testInstanciation()
```

Comment:
*Instantiation

Reply:
Sorry, I used the French word... :man_shrugging: